plugin/src/Reconciler.lua

```diff
@@ -27,6 +28,29 @@ function Reconciler:applyUpdate(requestedIds, virtualInstancesById)
 	end
 end
 
+function Reconciler:mapProperties(virtualProperties, virtualInstancesById)
+	-- Maps properties from the Rojo format to one rbx-dom-lua uses
+	local properties = {}
+
+	for propertyName, property in pairs(virtualProperties) do
+		if property.Type == "BinaryString" then
+			local binaryString = ""
+			for _, characterCode in pairs(property.Value) do
+				binaryString = binaryString .. string.char(characterCode)
+			end
+			properties[propertyName] = binaryString
```

Comment:
Hmm, this looks like something to fix in the way `rbx_dom_weak` is serialized/deserialized! Maybe we can switch to base64 encoding along the lines of how the XML format works, and then keep them as strings.

Reply:
And then what, decode from rbx_dom_lua?